### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by olawale-o**
## In this PR, I did the following:

- Built a SPA with React and Redux
- Retrieved data from API and stored in Redux store
- Create routes for every view
- Wrote units for redux reducers pure functions
- Wrote integration test for whole app

### Design
Project is styled according to the given these design guidelines:
- Colors (select one main color for your website).
- Typography: font face, size and weight.
- Layout: composition and space between elements.

### Documentation

Netlify- [Netlify](https://kind-goodall-4886a1.netlify.app/)
Loom - [Loom](https://www.loom.com/share/60196b2d5f5c4887a5836b8deee9c86e)

### CODE REVIEW

File: src/redux/stocks/stocks.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+const initialState = {
+  items: [],
+  total: 0,
+  selected: null,
+  loading: false,
+};
+
+// action.payload
+// { ...state, selected: action.payload };
+
```
Comment (Dipeshtwis) [inline src/redux/stocks/stocks.js:92]: - [ ] I would invite you to please remove the commented code as removing commented code makes the codebase cleaner.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @olawale-o, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Deployed the application using Netlify and working as expected.
:+1: The App used redux to store the data. Well Done.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/stocks/stocks.js (hunk 1 above)
Comment (olawale-o) [reply src/redux/stocks/stocks.js:92]: Hi @Dipeshtwis

Thank you for your time, effort and for pointing out these changes in this review

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @olawale-o,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by olawale-o** [comment]
Hi @Dipeshtwis 
Thank you for your time and effort on the review. Keep up the good work

**Pull request merged**